### ISSUE TITLE
Can't find package

### ISSUE THREAD

**Issue by mindyourlifeguide** [nice-registry/all-the-package-names#19]
I tried to find the [effector-logger](https://github.com/sergeysova/effector-logger) package through it and did not find it, although there is such a package

**Comment by zeke** [comment]
Yeah this project is a bit out of date right now. The automation broke and I have not prioritized fixing it. Help would be welcome!
